docs/topics/whatsnew1820.md

```diff
@@ -409,7 +409,7 @@ We want to highlight the following advantages of the new Kotlin/Wasm target:
   easy-to-parse bytecode.
 * Improved application runtime performance compared to Kotlin/JS and JavaScript because Wasm is a statically typed language.
 
-Starting with the 1.8.20-RC2 preview release, you can use Kotlin/Wasm in your experimental projects.
+Starting with the 1.8.20 release, you can use Kotlin/Wasm in your experimental projects.
```

Comment:
JFYI
I think it's not a `preview release` anymore

Reply:
very true - thank you for noticing.